master/reference/calicoctl/resources/profile.md

```diff
@@ -2,116 +2,87 @@
 title: Profile Resource (profile)
 ---
 
-A Profile resource (profile) can be thought of as describing the properties of 
-an endpoint (virtual interface, or bare metal interface).  Each endpoint can 
-reference zero or more profiles.  A profile encapsulates a specific set of tags,
-labels and ACL rules that are directly applied to the endpoint.  Depending on 
-the use case, profiles may be sufficient to express all policy.
+A Profile resource (profile) represents a set of rules which are applied applied
+to a collection of endpoints that have been assigned to the profile.
 
-Also see the [Policy resource]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/profile) 
+Each Calico endpoint or host endpoint can be assigned to zero or more profiles.
+
+Also see the [Policy resource]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/policy) 
 which provides an alternate way to select what policy is applied to an endpoint.
 
 ### Sample YAML
 
-```
+The following sample profile allows all traffic from 10.0.0.0/16, except if the source 
+is within 10.244.0.0/24.
+
+```yaml
 apiVersion: v1
 kind: profile
 metadata:
   name: profile1
   labels:
-   foo: bar
+    profile: profile1 
 spec:
-  tags:
-  - tag1
-  - tag2s
   ingress:
   - action: deny
-    protocol: tcp
-    icmp:
-       type: 10
-       code: 6
-    notProtocol: udp
-    notICMP:
-       type: 19
-       code: 255
     source:
-      tag: production
-      net: 10.0.0.0/16
-      selector: type=='application'
-      ports: 
-      - 1234
-      - "10:20"
-      notTag: bartag
-      notNet: 10.1.0.0/16
-      notSelector: type=='database'
-      notPorts: 
-      - 1050
-    destination:
-      tag: alphatag
-      net: 10.2.0.0/16
-      selector: type=='application'
-      ports: 
-      - "100:200"
-      notTag: bananas
-      notNet: 10.3.0.0/16
-      notSelector: type=='apples'
-      notPorts: 
-      - "105:110"
-  egress:
+      net: 10.244.0.0/24
```

Comment:
The idea is that the example shouldn't need to be a complicated one - the reference below should explain rather than people looking to some sample json.

The sample is just a starting point, and I think this one is more realistic than one with a whole bunch more fields defined.

In general, I think users will only use a small subset of the supported fields in any given profile.